spec/acceptance/realtime/client_spec.rb

```diff
@@ -142,7 +142,7 @@
             context 'and an explicit client_id in ClientOptions' do
               let(:client_id) { random_str }
 
-              it 'allows uses the explicit client_id in the connection' do
+              xit 'allows uses the explicit client_id in the connection' do
```

Comment:
Could you add comment for skipped tests that they will be fixed as a part of https://github.com/ably/ably-ruby/issues/425? (I think skipped test always should be with comment describing why they were skipped and when we are going to fix them)

Reply:
Sure